typescript/db/schema.prisma

```diff
@@ -63,7 +63,7 @@ model Dataset {
   workspaceSlug String
   workspace     Workspace    @relation(fields: [workspaceSlug], references: [slug], onDelete: Cascade)
 
-  @@unique(fields: [workspaceSlug, slug], name: "workspaceSlugAndDatasetSlug")
+  @@unique(fields: [workspaceSlug, slug], name: "slugs")
```

Comment:
Last time, @crubier asked to name this `workspaceSlugAndDatasetSlug` to be consistent with `workspaceSlugAndDatasetName`. Was he ok to do this change?

Reply:
If we leave `workspaceSlugAndDatasetSlug ` then we can't do this when querying Prisma:

```gql
query dataset{
  dataset(where:{slugs:{slug:"my-first-dataset", workspaceSlug:"carnicero"}}){
    id
    name
  }
}
```

Instead, we have to do this:

```gql
query dataset{
  dataset(where:{workspaceSlugAndDatasetSlug:{slug:"my-first-dataset", workspaceSlug:"carnicero"}}){
    id
    name
  }
}
```

So I went ahead and changed the name since we agreed it was nicer in the API to have it this way. We can move it back to `workspaceSlugAndDatasetSlug` but it would mean that we need to change all the queries for dataset from slugs to the above query (even the local ones)